test/Microsoft.SqlTools.ManagedBatchParser.IntegrationTests/BatchParser/BatchParserTests.cs

```diff
@@ -232,15 +230,52 @@ public void VerifyLexerSetState()
                 }
                 executionResult = ScriptExecutionResult.Success;
             }
-            catch (Exception ex)
+            catch (Exception)
             {
-
                 executionResult = ScriptExecutionResult.Failure;
-
             }
             //  we doesn't expect any exception or testCase failures
             Assert.Equal<ScriptExecutionResult>(ScriptExecutionResult.Success, executionResult);
-             
+        }
+
+        // This test case is to verify that, Powershell's Invoke-SqlCmd handles ":!!if" in an inconsistent way
+        // Inconsistent way means, instead of throwing an exception as "Command Execute is not supported." it was throwing "Incorrect syntax near ':'."
+        [Fact]
+        public void VerifySqlCmdExecute()
```

Comment:
This is a really terrible name: it is not descriptive at all of that the test does.